src/color_wheel.cpp

```diff
@@ -571,4 +667,33 @@ void ColorWheel::dropEvent(QDropEvent* event)
     }
 }
 
+void ColorWheel::clearHarmonies()
+{
+    p->ring_editors.clear();
+    p->current_ring_editor = -1;
+    Q_EMIT harmonyChanged();
+    update();
+}
+
+void ColorWheel::addHarmony(double hue_diff, bool editable, int symmetric_to, int opposite_to)
```

Comment:
Yeah, you understand correctly.

The reason for this was poor performance when harmonies changed, but now that i look at it it doesn't make that much of a difference (at least for me). So maybe it would be better to just emit `harmonyChanged` and call `update` and get rid of `applyHarmonies`.